<launch>

    <node pkg="sensors" type="ins_2d" name="ins_2d" />

    <node pkg="rosserial_python" type="serial_node.py" name="serial_node" args = "/dev/ttyUSB1" />

    <node pkg="sensors" type="ardumotors.py" name="ardumotors" />

    <node pkg="sensors" type="waypoints3.py" name="waypoints3" />

    <node name="ddpg_guidance" pkg="sensors" type="ddpg_guidance.py" >
	<param name = "speed_equation" value = "0" />
    </node>

    <include file="$(find sensors)/launch/asmc.launch" />

    <!--Launch rosbag to record topics -->
    <node type="record" pkg="rosbag" name="recorder" args="record -a -o /home/ubuntu/catkin_ws/src/sensors/bags/ddpg/wo_speed/zz" />

</launch>
